src/test/groovy/nebula/plugin/dependencylock/DependencyLockAlignmentLauncherSpec.groovy

```diff
@@ -32,6 +32,9 @@ class DependencyLockAlignmentLauncherSpec extends BaseIntegrationTestKitSpec {
     def setup() {
         definePluginOutsideOfPluginBlock = true
         keepFiles = true
+
+        //TODO make task listener changes for config cache
+        disableConfigurationCache()
```

Comment:
I found that we need to take a look at the dependency verifier because I believe is using `TaskExecutionListener` and it is incompatible. While the build does not fail, the listener is not active it appears and the tests don't meet the conditions: an actual build failure